portal/index.php

```diff
@@ -51,8 +51,8 @@
     }
 
     if (count($result2) == 1) {
-        $defaultLangID = $result2[0]["lang_id"];
-        $defaultLangName = $result2[0]["lang_description"];
+        $defaultLangID = $result2[0]{"lang_id"};
+        $defaultLangName = $result2[0]{"lang_description"};
```

Comment:
just checking that you meant to do this (ie. fixing a bug). if not let me know, and I'll change these back (no need to revert for the 5.0.2 branch since this is just fixing a deprecated warning for php 7.4)

Reply:
No idea what that was so fix if needed thanks